build-logic/binary-compatibility/src/main/groovy/gradlebuild.binary-compatibility.gradle

```diff
@@ -47,8 +47,15 @@ repositories {
     }
 }
 
-def apiChangesJsonFile = layout.projectDirectory.file("src/changes/accepted-public-api-changes.json")
-def acceptedViolations = AcceptedApiChanges.parse(providers.fileContents(apiChangesJsonFile).asText.get())
+// We support multiple accepted public api changes JSON files, but "accepted-public-api-changes.json"
+// is the "main" one that is also shown in HTML report
+def apiChangesJsonDirectory = layout.projectDirectory.dir("src/changes/accepted-changes")
```

Comment:
💭 We now have `src/changes/accepted-changes` where any `.json` file is discovered